developers/development/guidelines.md

```diff
@@ -130,18 +130,22 @@ void myFun() {
 }
 ```
 
-* Where ever exceptions are caught and logged, the exception should be added as a last parameter to the logging. 
+* Decide on a case by case base, if you want exception stack traces in the log.
```

Comment:
I think you can replace this with...

Exceptions with stacktraces in the log are considered to be bugs in your binding that should be reported and fixed. 
If you add an exception as a last parameter to the logging, the stack trace will be printed. 
Configuration errors by users should only print log messages about what's wrong. In that case you would use `e.getMessage()`.